build-system/tasks/coverage-map/coverage-map.js

```diff
@@ -0,0 +1,111 @@
+/**
```

Comment:
Rename file to `index.js`. It will be `coverage-map/index.js`. All the downstream code can continue to refer to it as `coverage-map`. E.g. `const coverageMap = require('.../coverage-map')`.

Reply:
Done.